internal/storage/fake_storage_util.go

```diff
@@ -26,7 +26,20 @@ const TestObjectName string = "gcsfuse/default.txt"
 const ContentInTestObject string = "Hello GCSFuse!!!"
 const TestObjectGeneration int64 = 780
 
-func GetTestFakeStorageObject() fakestorage.Object {
+type fakeStorage interface {
```

Comment:
interface should always be public. Why is interface not public and struct is public?

Reply:
fixed